tests/cases/conformance/types/tuple/variadicTuples2.ts

```diff
@@ -11,6 +11,11 @@ type V10 = [number, ...string[], ...boolean[]];  // Error
 type V11 = [number, ...string[], boolean?];  // Error
 type V12 = [number, string?, boolean];  // Error
 
+type V15 = [...string[], ...number[]];  // Error
+type V16 = [...string[], ...Array<number>];  // Error
+type V17 = [...Array<string>, ...number[]];  // Error
+type V18 = [...Array<string>, ...Array<number>];  // Error

```

Comment:
I propose some extra test cases with type aliases:
```suggestion
type Alias19 = number[];
type V19 = [...string[], ...Alias19];  // Error
type Alias20 = Array<number>;
type V20 = [...string[], ...Alias20];  // Error
```